src/test/java/net/bramp/ffmpeg/FFprobeTest.java

```diff
@@ -57,6 +58,13 @@ public void testVersion() throws Exception {
   }
 
   @Test
+  public void testStartPtsType()
```

Comment:
Unless we need the real file, just use the fake fixture. It'll keep the repo file size down, and avoids copyright issues :-)

Appreciate the effort